tools/ja-rule/ja-rule.cpp

```diff
@@ -292,80 +296,69 @@ class Controller {
          << ". Use +/- to adjust, Enter commits, Esc to abort" << endl;
   }
 
-  void EchoCommandComplete(
-      ola::usb::USBCommandResult result,
-      uint8_t return_code, uint8_t status_flags,
-      const ByteString &payload) {
+  void EchoCommandComplete(ola::usb::USBCommandResult result,
+                           JaRuleReturnCode return_code, uint8_t status_flags,
+                           const ByteString &payload) {
     if (!CheckResult(result, status_flags)) {
       return;
     }
 
-    if (return_code != ola::usb::COMMAND_RESULT_OK) {
-      OLA_INFO << "RC: " << static_cast<int>(return_code);
-      return;
-    }
-
     string response;
     if (!payload.empty()) {
        response.append(reinterpret_cast<const char*>(payload.data()),
                        payload.size());
     }
-    OLA_INFO << "Echo Reply (" << static_cast<int>(return_code) << "): "
```

Comment:
I'm meaning more so that the output is actual useful/friendly for a human, but you'll see more in my upcoming PR.